index.bs

```diff
@@ -423,6 +418,16 @@ To perform a <dfn abstract-op>cross-origin resource policy check</dfn> given a [
         error-handling behavior.
 
 
+Integration with Service Worker {#integration-sw}
+-------------------------------------------------
+
+In https://w3c.github.io/ServiceWorker/#dom-fetchevent-respondwith, insert the following item
```

Comment:
Can you retain the explanation from the issue deleted above? That is, it's helpful to explain _why_ we're doing this, not just _what_ we're doing. :)

Reply:
Done.